amr-wind/wind_energy/ABLStats.cpp

```diff
@@ -429,9 +440,85 @@ void ABLStats::write_netcdf()
                 var.put(l_vec.data(), start, count);
             }
         }
+
+        {
+            amrex::Vector<std::string> var_names{
+                "u'v'_sfs", "u'w'_sfs", "v'w'_sfs"};
+            for (int i = 0; i < AMREX_SPACEDIM; i++) {
+                m_pa_sfs->line_average(i, l_vec);
+                auto var = grp.var(var_names[i]);
+                var.put(l_vec.data(), start, count);
+            }
+        }
+
+        {
+            amrex::Vector<std::string> var_names{
+                "u'theta'_sfs", "v'theta'_sfs", "w'theta'_sfs"};
+            for (int i = 0; i < AMREX_SPACEDIM; i++) {
+                m_pa_tsfs->line_average(i, l_vec);
+                auto var = grp.var(var_names[i]);
+                var.put(l_vec.data(), start, count);
+            }
+        }
     }
     ncf.close();
 #endif
 }
 
+void ABLStats::calc_sfs_stress_avgs()
```

Comment:
I would rename this `output_sfs_stress_avgs` and move the above section down here. That way you don't have to store instances of those averaging classes.

Reply:
Tried addressing in https://github.com/Exawind/amr-wind/pull/290/commits/22cbdfe875f7c910a3fd1369c871a361c06ac0dd